packages/block-editor/src/components/link-control/style.scss

```diff
@@ -61,6 +61,10 @@ $preview-image-height: 140px;
 .block-editor-link-control__field {
 	margin: $grid-unit-20; // allow margin collapse for vertical spacing.
 
+	.components-base-control__label {
+		color: $gray-900;
+	}
```

Comment:
Adding style overrides is usually not the best way to fix these issues:

- If `BaseControl`'s default label color is wrong, maybe the fix should happen directly in `BaseControl`
- Otherwise, if `BaseControl`'s default label color is correct, but the color is wrong specifically for `LinkControl`, then we should find which other CSS rule is changing the color, and work on a fix at that level
- We shouldn't be using internal `.components-` classnames, since those are meant to be implementation details of those components.

Any chance we can try to solve this issue in an alternative way?